apps/site/lib/site_web/plugs/retirement_message.ex

```diff
@@ -0,0 +1,31 @@
+defmodule SiteWeb.Plugs.RetirementMessage do
```

Comment:
Since the content of this plug is only applicable to retiring the Schedule Tab, could we rename this to be more specific? Something like `SiteWeb.Plugs.ScheduleTabRetirementMessage`?

(I'm also noting here that this plug doesn't seem quite suited for more generalized writing of retirement messages for other things that might come up in the future - do correct me if that was actually your intent!)

Reply:
The whole point of creating a plug (besides learning to build one for the first time) was to actually have a reusable message for the future by having a header and a body.

What disadvantages do you see? You mentioned a view function would suffice but I'm curious about scenarios in which a plug would be a better fit than here.

Also let me know if you prefer said view function and I can do that instead.

Reply:
Hmm. I think if you adjust the plug's call function to be able to return different messages based on what's in the `conn` (like I mentioned in a previous comment) that'll give us that reusability. With the pattern matching approach I'm envisioning it could even support multiple retirement messages simultaneously across the site 🤩